pkg/geo/geos/geos.cc

```diff
@@ -1160,6 +1173,66 @@ CR_GEOS_Status CR_GEOS_HausdorffDistanceDensify(CR_GEOS* lib, CR_GEOS_Slice a, C
   return toGEOSString(error.data(), error.length());
 }
 
+//
+// PreparedGeometry
+//
+
+CR_GEOS_Status CR_GEOS_PrepareGeometry(CR_GEOS* lib, CR_GEOS_Slice a, CR_GEOS_PreparedGeometry* ret) {
```

Comment:
can we call this `CR_GEOS_Prepare` to mimic the GEOS name

Reply:
Sure, changed